joomla/joomla-cms
# WIP:autoloading router fix

Don't use autoloader if you want to include 'router.php' manually

### Summary of Changes
Excluding router autoloader as Joomla have no in itself.

### Testing Instructions

Issues can appear on case insensitive file systems as autoloader can try to load 'Router.php' according to passed 'ComponentRouter'.
If file would be found but without class in it there is gonna be Fatal error as Joomla requires 'router.php'.
And on case insensitive system it will lead to file loaded twice.
